apps/evm/src/clients/api/queries/getLegacyPool/formatToPool/index.ts

```diff
@@ -75,18 +55,13 @@ export const formatToPool = ({
 }: FormatToPoolInput) => {
   const assets: Asset[] = [];
 
-  vTokenMetaDataResults.forEach((vTokenMetaData, index) => {
-    // Remove unlisted tokens
-    if (!vTokenMetaData.isListed) {
-      return;
-    }
-
+  legacyPoolMarkets.forEach((legacyPoolMarket, index) => {
     const underlyingTokenAddress =
       // If underlying asset address is the null address, this means the VToken has no underlying
       // token because it is a native token
-      areAddressesEqual(vTokenMetaData.underlyingAssetAddress, NULL_ADDRESS)
+      areAddressesEqual(legacyPoolMarket.underlyingTokenAddress, NULL_ADDRESS)
         ? NATIVE_TOKEN_ADDRESS
-        : vTokenMetaData.underlyingAssetAddress;
+        : legacyPoolMarket.underlyingTokenAddress;
```

Comment:
This logic isn't needed anymore since it's present here already: https://github.com/VenusProtocol/venus-protocol-interface/pull/3156/files#diff-c9a8841d55105945315df70e596199b85a70aa24bedaf507f506ae8b69a6217cR20